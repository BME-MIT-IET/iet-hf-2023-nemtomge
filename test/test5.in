loadmap map.json 
start V1 
set Gencode LA1 Immunity 10 10
set Scientist V1 PL1 
move V1 LA1 
show Gencode V1
touch V1
1
show Gencode V1
exit
