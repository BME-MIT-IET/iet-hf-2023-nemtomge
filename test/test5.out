Map from file map.json loaded
The game has been started. Players are: V1 
Genetic code Immunity aminoacid: 10 nucleotide: 10 has been placed on field LA1 
Scientist V1 set on field PL1 
Scientist V1 moved to field LA1 
V1 doesn't have any genetic codes
Choose one to pick: 
0 - cancel
1 - Immunity(3)
V1's genetic codes are:
Immunity(3)
